src/MyComponent.tsx

```diff
@@ -0,0 +1,240 @@
+import { useState, useRef, useMemo } from "react";
+import { useWindowSize } from "usehooks-ts";
+
+import './MyComponent.css';
+
+export default function MyComponent() {
+  const { width: windowWidth } = useWindowSize();
+
+  const [theme, setTheme] = useState<string>("Light");
+  const [visibleWork, setVisibleWork] = useState<boolean>(true);
+  const [visibleBrand, setVisibleBrand] = useState<boolean>(true);
+
+  const contentRef = useRef<HTMLDivElement>(null);
+  const imageHeight = useMemo(() => contentRef.current?.clientHeight, [windowWidth])
+
+  const handleReadLessButton = () => {
+    setVisibleWork(!visibleWork);
+  }
+
+  const handleReadMoreButton = () => {
+    setVisibleBrand(!visibleBrand);
+  }
+
+  return (
+    <div className="flex flex-col items-start py-16 pr-20 pl-8 bg-white max-w-[810px] max-md:px-5 font-inter">
+      <div style={{ background: theme === 'Light' ? '#fff' : '#333', color: theme === 'Light' ? '#333' : '#fff' }} className="py-20 px-20">
+        <div className="flex gap-0 justify-center self-center text-xl text-white whitespace-nowrap">
+          <div className={`${theme === 'Light' ? 'justify-center px-7 py-1.5 bg-blue-400 max-md:px-5' : 'justify-center px-7 py-1.5 bg-gray-400 max-md:px-5'}`}>
+            <button onClick={() => setTheme("Light")}>Light</button>
+          </div>
+          <div className={`${theme === 'Light' ? 'justify-center px-7 py-1.5 bg-zinc-800 max-md:px-5' : 'justify-center px-7 py-1.5 bg-blue-400 max-md:px-5'}`}>
+            <button onClick={() => setTheme("Dark")}>Dark</button>
+          </div>
+        </div>
+        <div className="flex gap-5 justify-between mt-8 w-full max-md:flex-wrap max-md:max-w-full">
+          {(theme === 'Light') &&
+            <img
+              loading="lazy"
+              src="https://cdn.builder.io/api/v1/image/assets/TEMP/ff40052dc9774bde4716c49009cc482a076b466d6da3c65944e71e8ac4135228?"
+              className="shrink-0 max-w-full aspect-[5.56] w-[209px]"
+            />
```

Comment:
Should use `alt` props for all of the image tags.

Reply:
Fixed.